doc/quickstart/configure.rst

```diff
@@ -62,6 +62,10 @@ omitted in the file.
   # on a computer without an internet connection.
   offline: true
 
+  # Search ESGF for files even when files are available locally. This option is
+  # useful to make sure you have the latest version of all files.
```

Comment:
It would be great if you could use the same syntax here as for other options with a fixed set of possible values, e.g., for `offline`:

```yaml
# Disable automatic downloads --- [true]/false
# Disable the automatic download of missing CMIP3, CMIP5, CMIP6, CORDEX,
# and obs4MIPs data from ESGF by default. This is useful if you are working
# on a computer without an internet connection.
```

Also, if I understand the code correctly this only works if `offline=False`. This may be obvious, but would be nice to add here:

So maybe something like

```suggestion
  # Search ESGF for files even when files are available locally --- true/[false]
  # This option is useful to make sure you have the latest version of all files.
  # Remember to set ``offline: false`` if this option is set to ``true``.
```